apps/invoices/src/lib/saleor-version-compatibility-validator.ts

```diff
@@ -0,0 +1,18 @@
+const semver = require("semver");
+
+/**
+ * TODO Extract to shared or even SDK
+ */
+export class SaleorVersionCompatibilityValidator {
+  constructor(private appRequiredVersion: string) {}
+
+  validateOrThrow(saleorVersion: string) {
+    const versionIsValid = semver.satisfies(saleorVersion, this.appRequiredVersion, {
+      includePrerelease: true,
```

Comment:
@przlada I replaced coerce with includePrerelease. It works, but I need to specify full Saleor version (`x.y.z`). Can you confirm Saleor *always* returns all three segments? It will fail if it returns `3.13` only (coercion is making it `3.13.0`)